test/server/cards/03_TWI/units/DaughterOfDathomir.spec.ts

```diff
@@ -0,0 +1,35 @@
+describe('Daughter of Dathomir', function () {
+    integration(function (contextRef) {
+        describe('Daughter of Dathomir\'s ability', function () {
+            it('should gain Restore 2 while undamaged', function () {
```

Comment:
should also have a test for when it is damaged (to make sure it doesn't restore)

Reply:
Oh, my bad. I was expecting separate tests and didn't see you combined into one.